include/git2/common.h

```diff
@@ -205,7 +205,8 @@ typedef enum {
 	GIT_OPT_SET_ALLOCATOR,
 	GIT_OPT_ENABLE_UNSAVED_INDEX_SAFETY,
 	GIT_OPT_GET_PACK_MAX_OBJECTS,
-	GIT_OPT_SET_PACK_MAX_OBJECTS
+	GIT_OPT_SET_PACK_MAX_OBJECTS,
+	GIT_OPT_IGNORE_PACK_KEEP_FILE_CHECK
```

Comment:
`GIT_OPT_DISABLE_PACKFILE_KEEP_CHECK/S` maybe ? I'd like to be consistent in what verbiage we use to enable/check/set/ignore…

Also, adding some documentation to the monster just below would be ❤️.

> * opts($name, …)
> This will cause `.keep` file existence checks to be skipped when accessing packfiles,
> which can help performance with remote filesystems.